config.md

```diff
@@ -244,7 +244,7 @@ This specification uses the following terms:
             Implementations MUST generate an error if they encounter a unknown value while verifying or unpacking an image.
           </li>
           <li>
-            <code>diff_ids</code> is an array of layer content hashes (<code>DiffIDs</code>), in order from bottom-most to top-most.
+            <code>diff_ids</code> is an array of layer content hashes (<code>DiffIDs</code>), in order from bottom-most to top-most and bottom-most at index 0.
```

Comment:
@wking Be very careful: I have explained the concept of `diff_ids` several times and you keep keeping it _absolutely wrong_. I would prefer if you could take the time to understand the difference before commenting further on this matter. You are causing a lot of confusion and wasting a lot of time of the maintainers and contributors.

This literally has nothing to do with #316, #318 or descriptors. The CAS system has no meaning when inside the image config. This is for calculating the secure ChainID which is used as the image id.